packages/waku/src/router/client.ts

```diff
@@ -196,16 +195,16 @@ export function useRouter_UNSTABLE() {
 
 export type LinkProps = {
   to: InferredPaths;
-  pending?: ReactNode;
-  notPending?: ReactNode;
+  children: ReactNode;
   /**
    * indicates if the link should scroll or not on navigation
    * - `true`: always scroll
    * - `false`: never scroll
    * - `undefined`: scroll on path change (not on searchParams change)
    */
   scroll?: boolean;
-  children: ReactNode;
+  unstable_pending?: ReactNode;
+  unstable_notPending?: ReactNode;
```

Comment:
This change is technically breaking change. I wasn't confident with this, so marking as unstable.